docs/tasks.md

```diff
@@ -153,3 +139,177 @@ work has completed, spinning up replacement workers.
 ```
 awx-manage run_dispatcher --reload
 ```
+
+* * *
+
+In the following sections, we will go further into the details regarding AWX/Tower tasks.  They are all decorated by `@task()` in [awx/awx/main/tasks.py](https://github.com/ansible/awx/blob/devel/awx/main/tasks.py)
+
+## Housekeeping Tasks
+
+The two main types of "housekeeping tasks" are _Tower Schedulers_ and _Task Managers_.  
```

Comment:
There's a bit of confusing terminology here, and it's our fault because we use a lot of terms interchangeably :(

I also think this section is a bit too in-depth, and a lot of the explanation already exists in https://github.com/ansible/awx/blob/devel/docs/task_manager_system.md#hybrid-scheduler-periodic--event

It might be better to have something succinct like:

> Tower includes a sophisticated system for scheduling jobs that are launched on demand or at scheduled times, primarily via the `run_task_manager` task.
> 
> For further information regarding Tower Schedulers or Task Managers, refer to the [Task Manager Overview page](https://github.com/ansible/awx/blob/devel/docs/task_manager_system.md) of the AWX documentation.